### ISSUE TITLE
[Feature Request]: Combine Account URL and Service URL open with preferences

### ISSUE THREAD

**Issue by nmat** [wavebox/waveboxapp#828]
* Wavebox Version: 3.4.1beta

**Is your feature request related to a problem? Please describe.**
At the moment it can be a bit confusing how to open links inside wavebox. Not everyone knows the diffrence between account profile and service. Or if one of the other opens link in a new window or running tab.

**Describe the solution you'd like**
*Preferences (open link in service):*
  Standard - Service Tab/Service New Window

On standard install the service link opens in window by default using credentials

On context menu:
Open in Service -> [service] -> Service Tab/New Window

(On [service] click follow preferences)

**Additional info**
Above is to cover both new window and current running service tab open preferences. On right click you can chose how to open the link but also be able to set standard setting for handling the link.

**Comment by Thomas101** [comment]
Hey, I Just want to check that I'm following you correctly on this one...

1. **Open link in Wavebox**
Leave this the same
![screen shot 2018-10-15 at 09 15 02](https://user-images.githubusercontent.com/103586/46938373-87939e80-d05b-11e8-858a-8c852d151aa1.png)

2. **Open link in service**
Remove this
![3](https://user-images.githubusercontent.com/103586/46939170-882d3480-d05d-11e8-9778-ee84ea9e56e4.png)

3. **Open link in account profile**
By default, keep the current behaviour. Have a setting that makes it do what `Open link in service` currently does?
![2](https://user-images.githubusercontent.com/103586/46939151-7cda0900-d05d-11e8-9b85-d95e335ce6d9.png)

**Comment by nmat** [comment]
Yes exactly. But have the option to open in new window or in service tab.

**Comment by nmat** [comment]
the functionality that is there now is good.

However its just the general Right click -> [confusing 2 menu options] that is keeping me at a loss for a bit. Since those "could" be combined in the way to say that you in the context menu open it in the tab or using the account profile (new window vs tab for service)

Example:
1. Zendesk: Right click on the link, go to context menu open in account -> Account -> Current running service/new window with the account details

**Comment by nmat** [comment]
Does this image make sense? (since it might be hard to read when hitting Ctrl or something to show the menu to overwrite the option in preferences)

![image](https://user-images.githubusercontent.com/383031/46982172-8609bb00-d0db-11e8-90b4-d3aa8d7b553f.png)

**Comment by nmat** [comment]
In the below gif you will see that I use total commander to show the first right click menu to get "standard" settings from right click menu.

On second right click I hold Alt+Shift then right click to show "more options" you can see then that *Open in powershell* etc is shown. That was the first idea I had for the service menu on holding Ctrl+Mouse over Google Drive for example to override the preference that is global.

In chrome for example I think you can use middle mouse button and it is hidden to open link in new tab. But like I also said I am open to ideas to make the menu easier to understand. :)

![2018-10-16_00-41-30](https://user-images.githubusercontent.com/383031/46982383-54452400-d0dc-11e8-9d96-d569c565ddc1.gif)

**Comment by Thomas101** [comment]
Thanks, that's a lot clearer now. I'll have a look and see what we can do!

**Comment by Thomas101** [comment]
Long term, I think the <kbd>shift</kbd> option for extra tools is the best way to go here. There's some investigation work we need to do around listening for keyboard events as the popout menu itself doesn't emit keyboard events but does gobble them up. This is probably a PR upstream on electron or chromium and could even depend on platform support.

The way it works at the moment makes it confusing. The difference between `Account Profile` and `Service` just isn't clear enough - even I can't remember the difference at a quick glance! We spoke at some length about how to name them and couldn't come up with anything that instantly stuck out. Because of this I'm not overly keen on pushing this out to stable in its current encarnacion.

What I've done, is change the way it works slightly to hopefully make it a little less confusing and then put the functionality under an advanced & experimental flag until we can hopefully get <kbd>shift</kbd> working...

![1](https://user-images.githubusercontent.com/103586/47089119-f7e71f00-d217-11e8-93d2-6293ce8bd388.png)

The two menus are now collapsed into the main `Open in Account Profile` menu but with a sub-menu spanning out. It makes a little bit more long-winded and menu-y to get down to the end, but hopefully the clarity it gives makes up for this. You can also open up in sleeping accounts which wasn't available before (it will wake the account and load the url).
![screen shot 2018-10-17 at 14 05 55](https://user-images.githubusercontent.com/103586/47089143-07fefe80-d218-11e8-9046-7618d1d1f9b8.png)

I'm hoping this gives us a good stop-gap until we can get <kbd>shift</kbd> or something else working :-)

**Comment by nmat** [comment]
This is actually really good! Love it! Will definitely try it out more on beta release but tested just quickly here.

Also I agree its hard to know what is what when it comes to the naming. To share how I think or look at the accounts is that:

![image](https://user-images.githubusercontent.com/383031/47108630-7067d300-d24c-11e8-9d4d-c9109a6a0887.png)

The registered account I also see still as "Wavebox Account" since its the "Registered Account". This is only my 2 cents. However I agree to 100% with what you are saying. It is less cluttered now and makes a bit more sense for the "view" of how wavebox is showing the services/accounts right now.

And I believe that you have better ideas then me. :)

**Comment by Thomas101** [comment]
I've pushed the change out to stable - but will keep this issue open as there's more work we can do here! Thanks
